opencog/spacetime/octomap/TimeOctomap.h

```diff
@@ -81,23 +82,38 @@ struct TimeSlice
         map_tree.setNodeData(location, ato);
     }
 
+    void insert_atom(const T& ato){
+        temporal.push_back(ato);
```

Comment:
please fix the whitespace ... you've been doing this long enough to know whitespace is important.